package.json

```diff
@@ -61,7 +64,9 @@
         "start:dev": "yarn setup; LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config conf/webpack.config.js --mode development",
         "start:examples": "yarn setup; LANGUAGE=en-US REACT=true BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config conf/styleguide.config.js --mode development",
         "start:npm": "yarn setup; npm-run-all build:prod:npm build:dev:es",
-        "test": "BABEL_ENV=test NODE_ENV=test yarn jest"
+        "test": "BABEL_ENV=test NODE_ENV=test yarn jest",
+        "test:e2e": "npm-run-all -p -r start:examples cy:run",
```

Comment:
Wondering if we should rename `e2e` to `integration` OR `integration` to `e2e` at other places because I think there are subtle differences, again based on a recent doc i saw. @djordan correct me if I am wrong, integration testing ideally does not go beyond testing the stack (so front end only but multiple components/units) while e2e goes beyond that so full stack (FE + backend).

Now I think here we are doing e2e tests here, because we are using the examples page, which is actually making network requests to the API and loading real live data. So we are doing e2e tests?